101-media-services-create/azuredeploy.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Media Services Account."
+      }
+    },
+    "storageName": {
```

Comment:
Yeah, that is what i prefer also - just generate the storage account based on the Media Services account name. It's all going into a unique Resource Group anyways hopefully